lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -614,6 +614,37 @@ serve as complete descriptions of the runtime of a tool.
       <xs:element name="requirement" type="Requirement" minOccurs="0" maxOccurs="unbounded"/>
       <xs:element name="container" type="Container" minOccurs="0" maxOccurs="unbounded"/>
     </xs:sequence>
+    <xs:attribute name="cores_min" type="xs:float" use="optional">
+      <xs:documentation xml:lang="en"><![CDATA[
+Maximum reserved number of CPU cores if runtime allows it (unimplemented in Galaxy currently).
```

Comment:
```suggestion
Minimum reserved number of CPU cores, if runtime allows it (not yet implemented in Galaxy).
```

Reply:
I'm redoing this, I don't think we'd want to write javascript expressions within attributes.
I'm thinking of this:
```
    <requirements>
        <resource type="cores_min" type="float">1.1</resource>
        <resource type="ram_max" type="expression"><![CDATA[runtime.cores * 2]]></resource>
    </requirements>
```